components/eamxx/src/mct_coupling/atm_comp_mct.F90

```diff
@@ -142,9 +142,13 @@ subroutine atm_init_mct( EClock, cdata, x2a, a2x, NLFilename )
     !----------------------------------------------------------------------------
 
     ! Init the AD
+    call seq_timemgr_EClockGetData(EClock, curr_ymd=cur_ymd, curr_tod=cur_tod, start_ymd=case_start_ymd, start_tod=case_start_tod)
     call string_f2c(yaml_fname,yaml_fname_c)
     call string_f2c(trim(atm_log_fname),atm_log_fname_c)
-    call scream_create_atm_instance (mpicom_atm, ATM_ID, yaml_fname_c, atm_log_fname_c)
+    call scream_create_atm_instance (mpicom_atm, ATM_ID, yaml_fname_c, atm_log_fname_c, &
+                          INT(cur_ymd,kind=C_INT),  INT(cur_tod,kind=C_INT), &
+                          INT(case_start_ymd,kind=C_INT), INT(case_start_tod,kind=C_INT))
```

Comment:
Feel free to ignore, but would be nicer (imo) for the last 2 lines to stack under the input of the first.